package.json

```diff
@@ -4,14 +4,17 @@
   "private": true,
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.3",
+    "axios": "^1.3.5",
     "bootstrap": "^5.2.3",
+    "oidc-client": "^1.11.5",
```

Comment:
You can remove "oidc-client" and use just "oidc-client-ts" and "react-oidc-context"

Reply:
Forgot to remove a dependency, originally used this library but had strange problems with it